xlml/utils/metric.py

```diff
@@ -570,7 +570,13 @@ def get_gce_job_status(
       wait_task = current_dag.get_task(
           task_id=f"{benchmark_id}.provision.create_queued_resource.wait_for_ready_queued_resource"
       )
-    elif isinstance(task_test_config.accelerator, test_config.Gpu):
+    elif isinstance(task_test_config, test_config.PlainGpuTest):
+      # Test using the GKE GPU.
+      wait_task = current_dag.get_task(
+          task_id=f"{benchmark_id}.run_model.stream_logs"
```

Comment:
FYI, this does have a different semantic than `wait_for_ready_queued_resource` or `get_ip_address`. Those pertain only to whether the accelerator was created. `stream_log` also streams the running test's logs.

There's a fundamental conceptual difference: some tests create accelerators on demand. For the GKE test, the accelerator is already there and waiting for a job (or finishing another task first).

@RissyRan do you think that could cause any issues with how we interpret failures?

Reply:
I just updated a separate `get_gke_job_status` to handle this job flow instead of adding too many conditions in existing `get_..._status` functions.